src/slim-select/render.ts

```diff
@@ -265,6 +267,13 @@ export default class Render {
     values.classList.add(this.classes.values)
     main.appendChild(values)
 
+    // Creating the element that shows the number of selected items
```

Comment:
I see where you would think it would go here and its a good thought but the better spot would be in the renderMultipleValues function. After the part after it checks if selected length is 0 for whether or not to add the placeholder. Right after that you would check your selected length and if it exceeds your limit then in that if statement it would create the div and add it the the values container.